src/modules/item/export.ts

```diff
@@ -153,14 +154,14 @@ function buildItemEntityMetadata(collection: Collection, item: Item): StandardCa
     name: item.name,
     description: item.description,
     collectionAddress: collection.contractAddress!,
-    rarity: item.rarity!,
-    i18n: [{ code: 'en', text: item.name }],
+    rarity: (item.rarity! as unknown) as Rarity,
```

Comment:
Same question as [here](https://github.com/decentraland/builder-server/pull/470#discussion_r839875557) in the server. If we won't change anything there, dismiss!